docs/products/clickhouse/howto/connect-with-java.rst

```diff
@@ -0,0 +1,48 @@
+Connect to the Aiven for ClickHouse® service with Java
+======================================================
+
+Learn how to connect to your Aiven for ClickHouse® service with Java using the ClickHouse JDBC driver and the HTTPS port.
+
+Pre-requisites
+--------------
+
+* `Java 8 <https://www.java.com/en/download/>`_ or later
+* `ClickHouse JDBC driver <https://github.com/ClickHouse/clickhouse-jdbc/tree/master/clickhouse-jdbc>`_
+
+Identify connection information
+-------------------------------
+
+To run the code for connecting to your service, first identify values of the following variables:
+
+===========================     =======================================================================================
+Variable                        Description
+===========================     =======================================================================================
+``CLICKHOUSE_HTTPS_HOST``       ``Host`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``CLICKHOUSE_HTTPS_PORT``       ``Port`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**                 
+``CLICKHOUSE_USER``             ``User`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**           
+``CLICKHOUSE_PASSWORD``         ``Password`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**           
+===========================     =======================================================================================
+
+Connect to the service
+----------------------
+
```

Comment:
I would add a first step here:
```
1. Add the ClickHouse JDBC driver to your Maven dependencies:
#``` <= can't nest them in GitHub :)
    <dependency>
        <groupId>com.clickhouse</groupId>
        <artifactId>clickhouse-jdbc</artifactId>
        <version>0.3.2-patch11</version>
        <classifier>all</classifier>
        <exclusions>
            <exclusion>
                <groupId>*</groupId>
                <artifactId>*</artifactId>
            </exclusion>
        </exclusions>
    </dependency>
 #```
```